links/gql_http_link/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.10
+
+- add `headers` to `HttpLinkResponseContext`
```

Comment:
I still think that this should be included on a release commit.

Reply:
What do you mean, should I remove this change?

Reply:
My understanding is we [update the changelog in the change PR](https://github.com/gql-dart/gql/pull/114#issuecomment-640757142), so this is right. @klavs seems we should probably make a more detailed contributor doc.